docs/LowLevelInfo.md

```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-This documnent will contain loose information for various topis referring to
+This document will contain loose information for various topics referring to
 the SRT source code describing some cross-source analysis that would be
-unobvious for a source code reviewer. It's not a complete documentation of
+non-obvious for a source code reviewer. It's not a complete documentation of
```

Comment:
Not sure about this change: "unobvious" -> "non-obvious". @stevomatthews what do you think?

Reply:
"that would not be obvious for a source code reviewer" is the safe way to go!  ;-)

Reply:
@quink-black Please also apply this suggestion.

Reply:
Done. It's amazing I can learning English from this project : P